clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -1360,6 +1361,87 @@ describe("Consent overlay", () => {
         });
       });
 
+      describe("when GPC flag is found, and notices apply to GPC and acknowledge button is used", () => {
+        beforeEach(() => {
+          cy.on("window:before:load", (win) => {
+            // eslint-disable-next-line no-param-reassign
+            win.navigator.globalPrivacyControl = true;
+          });
+          cy.getCookie(CONSENT_COOKIE_NAME).should("not.exist");
+          stubConfig({
+            experience: {
+              privacy_notices: [
+                mockPrivacyNotice({
+                  title: "Marketing",
+                  notice_key: "marketing",
+                  consent_mechanism: ConsentMechanism.OPT_IN,
+                  data_uses: [
+                    "marketing.advertising.first_party.targeted",
+                    "marketing.advertising.third_party.targeted",
+                  ],
+                  default_preference: UserConsentPreference.OPT_OUT,
+                  id: "pri_marketing",
+                }),
+                mockPrivacyNotice({
+                  title: "Functional",
+                  notice_key: "functional",
+                  consent_mechanism: ConsentMechanism.OPT_IN,
+                  data_uses: ["personalize"],
+                  default_preference: UserConsentPreference.OPT_OUT,
+                  id: "pri_functional",
+                }),
+                mockPrivacyNotice({
+                  title: "Essential",
+                  notice_key: "essential",
+                  consent_mechanism: ConsentMechanism.NOTICE_ONLY,
+                  data_uses: ["essential"],
+                  default_preference: UserConsentPreference.ACKNOWLEDGE,
+                  id: "pri_essential",
+                }),
+              ],
+            },
+            experienceConfig: {
+              layer1_button_options: Layer1ButtonOption.ACKNOWLEDGE,
+            },
+          });
+        });
+
+        it("does not get overridden by banner acknowledge button", () => {
+          cy.get("@FidesUpdated")
+            .should("have.been.calledOnce")
+            .its("lastCall.args.0.detail.extraDetails.consentMethod")
+            .then((consentMethod) => {
+              expect(consentMethod).to.eql(ConsentMethod.GPC);
+            });
+          cy.get("div#fides-banner").within(() => {
+            cy.get("button").contains("OK").click();
```

Comment:
Can we also assert that the expected values are sent to the API and in the cookie?